features/frameworks/unity/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+target_include_directories(mbed-greentea
```

Comment:
Why not create `mbed-unity` and `mbed-utest`?
You can then clearly show the dependencies with `target_link_libraries()`.

Reply:
I also had the same question. I approved this as the user facing lib stays but we can fix internally how they are structured

Reply:
@hugueskamba I have created as one library because it all under greentea frameworks and `client + unity + utest` together provides greentea template and it will be used together in the test suite